### PR TITLE
chore(m365): add m365 tenant hint message

### PR DESCRIPTION
**Pull request by kuojianlu**
- add setting up dev environment to prerequisites in README
- show m365 tenant hint message when sideloading

image
image

### CODE REVIEW

File: packages/cli/src/cmds/preview/constants.ts [hunk 1]
```diff
@@ -150,3 +150,6 @@ export const installApp = {
       "Pop up Azure Portal for you to connect your bot to Outlook channel.",
   },
 };
+
+export const m365TenantHintMessage =
+  "WARN: Please note that after you enrolled your M365 tenant in Office 365 Target Release, it may take 3-5 days for the enrollment to take effect.";
```
Comment (MuyangAmigo) [inline packages/cli/src/cmds/preview/constants.ts:155]: Suggestion: `M365 tenant` -> `developer tenant`; `3-5 days` -> `couple days`.

**Comment by MuyangAmigo** [comment]
Can we also add a learn more button links to here? https://docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/prerequisites

Can include an additional text: `Please click Learn more button for more information about setting up dev environment for extending Teams apps across Microsoft 365.`

File: packages/cli/src/cmds/preview/constants.ts (hunk 1 above)
Comment (kuojianlu) [reply packages/cli/src/cmds/preview/constants.ts:155]: Updated.

**Comment by kuojianlu** [comment]
> Can we also add a learn more button links to here? https://docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/prerequisites
> 
> Can include an additional text: `Please click Learn more button for more information about setting up dev environment for extending Teams apps across Microsoft 365.`

Added.

**Review by MuyangAmigo — DISMISSED** [review]

**Review by swatDong — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Pull request merged**